about.md

```diff
@@ -4,4 +4,14 @@ title: About
 permalink: /about/
 ---
 
-HackIllinois is a 36 hour, student-ran collegiate hackathon that focuses on bringing the open source community and college students together.
+# What is HackIllinois?
+
+HackIllinois is a 36 hour, student-run collegiate hackathon that focuses on making the open source community more accessible to college students. HackIllinois happens in February every year and brings in over 1,100 students from around the country. 
+
+# How do Hackers at HackIllinois work on Open Source?
+
+The HackIllinois Staff works really hard to bring in mentors from different open source projects to help students get started on contributing to OSS. These mentors work alongside hackers to make pull requests, solve issues, and create features with popular open source software. 
```

Comment:
replace "really" with something else more formal. it's the only thing that seems to be out of place with the tone of the rest of the article